src/core/UserIconGroup/UserIconGroup.test.tsx

```diff
@@ -0,0 +1,190 @@
+  const { container } = render(
+    <UserIconGroup iconSize='medium'>
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+    </UserIconGroup>,
+  );
+
+  const userGroupContainer = container.querySelector(
+    '.iui-user-icon-list',
+  ) as HTMLElement;
+  expect(userGroupContainer.classList).toContain(`iui-stacked`);
+  expect(userGroupContainer.querySelector('.iui-user-icon-count')).toBeTruthy();
+});
+
+it('should render not stacked', () => {
+  const { container } = render(
+    <UserIconGroup iconSize='medium' stacked={false}>
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+    </UserIconGroup>,
+  );
+
+  const userGroupContainer = container.querySelector(
+    '.iui-user-icon-list',
+  ) as HTMLElement;
+  expect(userGroupContainer.classList).not.toContain(`iui-stacked`);
+});
```

Comment:
Need more test cases, for things like 99+ length, last icon props, etc.

`yarn test` should show you which lines were not covered

Reply:
Still need a test for `countIconProps`.